src/App.scss

```diff
@@ -0,0 +1,3 @@
+body {
+  background-color: gainsboro;
```

Comment:
Colour values should be kept in `colors.scss`

Reply:
I will do this when code will be correct